encourage stackexchange in issue templates (#5737)
MetaMask/metamask-extension: .github/ISSUE_TEMPLATE/bug-report.md

---

<!--
BEFORE SUBMITTING: 

> 1) Please search to make sure this issue has not been opened already

Reviewer: ```suggestion
1) Please search to make sure this issue hasn't already been opened
```

Reviewer: Er, on a second read this wording seems fine
